napari/utils/colormaps/colormap.py

```diff
@@ -230,3 +235,62 @@ def default_color(self):
         return self.color_dict.get(None, (0, 0, 0, 0))
         # we provided here default color for backward compatibility
         # if someone is using DirectLabelColormap directly, not through Label layer
+
+
+def cast_labels_to_minimum_type_auto(
+    data: np.ndarray, num_colors: int
+) -> np.ndarray:
+    """Perform modulo operation based on number of colors
+
+    Parameters
+    ----------
+    data : np.ndarray
+        Labels data to be casted.
+    num_colors : int
+        Number of unique colors in the data.
+
+    Returns
+    -------
+    np.ndarray
+        Casted labels data.
+    """
+    dtype = minimum_dtype_for_labels(num_colors + 1)
+
+    return _cast_labels_to_minimum_type_auto(data, num_colors, dtype)
+
+
+@numba.njit(parallel=True)
```

Comment:
Ah, I was trying to figure out why these lines weren't covered. When I had this situation in skan, I used NUMBA_DISABLE_JIT=1 when computing coverage. (Though I can't find any evidence of this now... 😅) What do you think @Czaki?

Reply:
?

Reply:
Disabling only for a given test leads to a crash. I do not want to disable it globally as it will have a big performance impact on the whole test suite...

Reply:
ok. I go into numba code and see how they test disabling jit in test and reproduce it in test